cmd/internal/collector/query.go

```diff
@@ -87,10 +87,13 @@ func BuildQuery(metric MetricDescription, cluster string) Query {
 		Filters: []Filter{filter},
 	}
 
-	groupBy := []string{}
-	for _, label := range metric.Labels {
-		groupBy = append(groupBy, "metric.label."+label.Key)
-	}
+        groupBy := []string{}
```

Comment:
Got the same feeling, `go fmt` seems right, but: @cotedm used spaces instead of tab. 
Debating about space vs tabs is not really productive, but I will change those to tabs in order to be consistent with the rest of the files.

Reply:
indentation: fixed

Reply:
whoops sorry about the format mismatch!